src/scenes/home/Home.js

```diff
@@ -1,10 +1,11 @@
 import React, { Component } from "react";
-import { View, TouchableOpacity, Image } from "react-native";
+import { View, TouchableOpacity, Image, Text } from "react-native";
 import ListTask from "./ListTasks";
 import styles from "./Home.styles";
 import colors from "src/helpers/Colors";
 import { Route } from "src/helpers/Route";
 import iconPlus from '../../../assets/iconPlus.png';
+import strings from "../../helpers/Strings";
```

Comment:
Please try to be consistent in the way you use imports _(relative/absolute)_.

I personally encourage you to use absolute ones.